api/config.py

```
@@ -69,10 +77,13 @@ def fhir_mcp_enabled(self) -> bool:
             and self.fhir_client_secret.strip()
             and self.fhir_token_url.strip()
             and self.notification_endpoint.strip()
+            and self.fhir_webhook_secret.strip()
         )
 
 
-def build_fhir_mcp_server(settings: Settings) -> FHIRMCPServer:
+def build_fhir_mcp_server(settings: Settings) -> "FHIRMCPServer":
```

Ruff (0.15.7)

[error] Undefined name `FHIRMCPServer`

(F821)